.husky/pre-commit

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-set -eo pipefail
+# set -eo pipefail
```

Comment:
It’s because in ubtunu `sh` is symlinked to `dash` which doesn’t support this option. Fix should be to replace the shebang on the first line with `#!/bin/bash`

Reply:
This didn't work for me, I got the  same error. I also tried `#!/usr/bin/bash` and other variants.

This works:
```
#!/bin/bash
set -e pipefail
```
According to a quick google this still fails early in pipelines, but I'm not sure. I can try and open up a new PR instead on my Mac, where this commit should just work (tm).